### DISCUSSION TITLE
ArchR multiome analysis: varFeature in scRNA-seq addIterativeLSI()

### DISCUSSION THREAD

**Discussion by Jiahuan90** [GreenleafLab/ArchR#1456] [Questions / Documentation] [answered]
Hi rcorces, 

I met a problem that I ran multiome ArchR pipeline scRNA-seq part with varFeature = 2500 successful.
proj <- addIterativeLSI(
    ArchRProj = proj,
    clusterParams = list(
      resolution = 0.2,
      sampleCells = 10000,
      n.start = 10
    ),
    saveIterations = FALSE,
    useMatrix = "GeneExpressionMatrix",
    depthCol = "Gex_nUMI",
    varFeatures = 2500,
    firstSelection = "variable",
    binarize = FALSE,
    name = "LSI_RNA"
)

However, when I adjust the varFeatures = 25000, it gave me error messages: "nFeature for variable selection must be at leat 1/2 the total features!". 

I don't know what is the meaning of varFeatures and how to set the rationale number for this parameter in scRNA-seq part? 2500? 5000? or something else? 

My second question is: should the varFeature parameters be same in scATAC-seq part? Since I use varFeature = 25000 in scATAC-seq part successful. 

Thank you for your kindly help!

**Comment by rcorces** [comment, marked as answer]
Sorry. That is a typo which I've corrected via https://github.com/GreenleafLab/ArchR/commit/21099d6ed966f6088b7b49ebbb9cca07486e8875

It should say "nFeature for variable selection must be less than 1/2 the total features"

You cannot have `varFeatures` be greater than half of the total features. Because ATAC has way more peaks than RNA has genes, you should not use the same values for `varFeatures` in both cases.

**Reply by Jiahuan90** [reply]
Thank you for your prompt and helpful response!
